wpeverest/everest-forms
# Fix - Focus on last error message on form validation

### All Submissions:






### Changes proposed in this Pull Request:



Previously, When form is submitted without filling required field then it focus last error only. This PR allow to focus on first first error.

### How to test the changes in this Pull Request:

1. Make form having required field more than 10.
2. Submit without filling. 
3.Check whether it focus on first error or not.

### Types of changes:




### Other information:




### Changelog entry

> Fix - Focus on last error message on form validation
